ros/src/fastrack/include/fastrack/planning/planner_manager.h

```diff
@@ -98,7 +98,7 @@ class PlannerManager : private Uncopyable {
   virtual void TimerCallback(const ros::TimerEvent& e);
 
   // Create and publish a marker at goal state. 
-  virtual void VisualizeGoal();
+  const virtual void VisualizeGoal();
```

Comment:
lolz nooooooo! I meant `virtual void VisualizeGoal() const;`

Reply:
lmao sorry -- will change!

Reply:
Also change the implementation signature to match.